.gitmodules

```diff
@@ -0,0 +1,73 @@
+# This is a git-fleximod adapted .gitmodules file.  Any field with a name starting in fx is a git-fleximod
+# specific field.  See https://github.com/ESMCI/git-fleximod for details.
+
+[submodule "cism"]
+path = source_cism
+url = https://github.com/ESCOMP/cism
+fxtag = cism_main_2.01.013
+fxrequired = AlwaysRequired
+fxDONOTUSEurl = https://github.com/ESCOMP/cism
+
+[submodule "cime"]
+path = cime
+url = https://github.com/ESMCI/cime.git
+fxtag = cime6.0.242
```

Comment:
So this tag updates externals and removes manage externals in one go. I'm ok with that but making a note because this is different than other plans I've heard.

Reply:
The cime external needed to be updated to fix an issue in ERI tests.   I think it's okay since it didn't change any answers.

Reply:
That's fine with me.